Library/Formula/vamp-plugin-sdk.rb

```diff
@@ -0,0 +1,24 @@
+
+class VampPluginSdk < Formula
+  homepage "http://www.vamp-plugins.org/"
+  url "http://sourceforge.net/projects/vamp/files/vamp-plugin-sdk/2.2.1/vamp-plugin-sdk-2.2.1.tar.gz/download"
+  version "2.2.1"
+  sha256 "571481098270133d2b78c6a461b850e04a98ab38284227c4d8056385f6333c26"
+
+  depends_on 'pkg-config' => :build
+  depends_on 'libsndfile'
+
+  def patches
```

Comment:
This is the output I get from the `patch do` block:

```
==> Patching
/usr/bin/patch -g 0 -f -p1 -i /private/tmp/vamp-plugin-sdk--patch20150421-49990-1kpu654/osx_settings
patching file Makefile.in
/usr/bin/patch -g 0 -f -p1 -i /private/tmp/vamp-plugin-sdk--patch20150421-49990-zjta5v/fix_library_paths
patching file Makefile.in
Hunk #1 FAILED at 102.
1 out of 1 hunk FAILED -- saving rejects to file Makefile.in.rej
Error: Failure while executing: /usr/bin/patch -g 0 -f -p1 -i /private/tmp/vamp-plugin-sdk--patch20150421-49990-zjta5v/fix_library_paths
```

If I use the `def patches` I don't get this error.